swagger/docs/css/print.css

```diff
@@ -0,0 +1,1172 @@
+/* Original style from softwaremaniacs.org (c) Ivan Sagalaev <Maniac@SoftwareManiacs.Org> */
```

Comment:
Can Swagger be used by adding a dependency or something, or does it have to be in our repo?